js/map.js

```diff
@@ -127,6 +128,7 @@ function addMarker(map, nodeResult, name, location) {
 	Description += '<h1>' + name + '</h1>';
 	Description += '<p>Status: ' + nodeStatus + '</p>';
 	if (nodeResult['cardinalDirection']) Description += '<p>Direction: ' + nodeResult['cardinalDirection'] + '</p>';
+	if (nodeResult['cardinalDirectionAntenna']) Description += '<p>Antenna: ' + nodeResult['cardinalDirectionAntenna'] + '</p>';	
```

Comment:
Would `Antenna Direction` work here? It seems more descriptive

Reply:
I was worried about length of box with added words but ipv6 add takes care of that already:)
corrected